src/backend/TrafficCourts/Citizen.Service/Models/Disputes/SearchDisputeResult.cs

```diff
@@ -1,17 +1,17 @@
-﻿using Swashbuckle.AspNetCore.Annotations;
-using System.ComponentModel.DataAnnotations;
+﻿using Swashbuckle.AspNetCore.Annotations;

```

Comment:
Using statements,

- Swashbuckle.AspNetCore.Annotations
- System.ComponentModel.DataAnnotations
- TrafficCourts.Messaging.MessageContracts

Are not needed

Reply:
done in upcoming commit